elm/config/elm_cfg.yaml

```diff
@@ -1,20 +1,22 @@
 model: codegen-350M-mono
 checkpoints_dir: ./checkpoints/
-device: cuda
+cuda: True
+gpus: 1
 seed: 42
 deterministic: False
+fp16: False
 top_p: 0.95
 temp: 0.85
-timeout: 5  # Seconds
-gpus: 1
-max_length: 1024
+timeout: 5.0  # Seconds
+gen_max_len: 1024
 batch_size: 1
-init_steps: 1000
-n_steps: 10000
-n_bins: 12
-fp16: False
+evo_init_steps: 1000
+evo_n_steps: 10000
+behavior_n_bins: 12
+evo_history_length: 10
+evaluation_steps: 1000
 pad_token: 50256
-task: "Sodarace"
+env_name: "sodarace"
 run_name: ???  # Mandatory string argument that describes the run.
```

Comment:
For this line (it's been like that since the beginning of ELM), every time I run I have to change ??? into something. I always forgot to ask but is this the expected thing to do? Or should we give it a name or something.